frontend/src/model/Tile.ts

```diff
@@ -0,0 +1,20 @@
+import Locations from '../constants/locations';
+
+class Tile {
```

Comment:
let's go one step further with this class - we need to add information about Tile specific locations (eg. monastery, garden) 

it would be nice if we implement 2 methods of Tile - rotateLeft, rotateRight - it should shift edges so it will be easier to validate Tile placement in the future

Reply:
Other ideas:
- placedBy which will be a reference to a Player class (it may be used for statistics in the future)
- placementTurn - it will be used to check if a user is eligible to place a meeple (eg. currentTurn === placementTurn)